src/pages/Sign/sections/SignSubscription.tsx

```diff
@@ -188,16 +189,45 @@ const StateComponent: React.SFC<StateComponentProps> = ({
     case SIGNSTATE.COMPLETED:
       if (collectStatus.status === BANKIDSTATUS.COMPLETE) {
         trackSignCompleted()
-        return (
-          <CurrentLanguage>
-            {({ currentLanguage }) => (
-              <Redirect
-                to={`/${currentLanguage &&
-                  currentLanguage + '/'}new-member/download`}
-              />
-            )}
-          </CurrentLanguage>
-        )
+
+        if (process.env.NODE_ENV !== 'test') {
+          return (
+            <OfferContainer>
+              {(offer) => {
+                adtraction(
```

Comment:
this part bugs me. Why are we doing this on every render? Is that really what was intended?